src/crimson/os/seastore/segment_manager/block.h

```diff
@@ -165,12 +139,29 @@ class BlockSegmentManager final : public SegmentManager {
     return superblock.segment_size;
   }
 
+  device_id_t get_device_id() const final {
+    return superblock.device_id;
+  }
+  secondary_device_set_t& get_secondary_devices() final {
```

Comment:
This shouldn't be part of the SegmentManager interface.  There's a call to get the superblock, that should suffice.

Reply:
Oh, there actually isn't.  Hmm, ok, we can leave this like this for now.